matlab_simulation/code v1.0/ME597-6-FastSLAM/newCode/callFS.m

```diff
@@ -0,0 +1,9 @@
+clear all; close all; clc;
```

Comment:
Some comments that explain the purpose of this file would be useful. Maybe I'm missing something, but I'm lost as to how this relates to the gyro covariance example.